css/style.css

```diff
@@ -458,14 +458,49 @@ button.button-inline:hover {
 	min-height: initial;
 }
 
+.card-controls .icon-filetype-text {
+	margin-right: 5px;
+}
+
 .due {
-	background-color: #eee;
-	color: #aaa;
 	padding: 1px 3px;
 	border-radius: 4px;
-	margin-right: 2px;
+	margin-right: 4px;
+	font-size: 90%;
+	opacity: .7;
+}
+.due .icon {
+	background-size: contain;
+	float:left;
+	opacity: 0.7;
+	margin-top:2px;
+}
+.overdue {
+	background-color: #e12419;
+	color: #fff;
+}
+.due.now {
+	background-color: #fbd850;
+	color: #000;
+}
+.due.next {
+	background-color: #22ac2a;
+	color: #fff;
 }
 
+.due .icon-calendar {
```

Comment:
you should move this over to `.due .icon` as it is a default setting. Other `.something .icon-calendar` declarations will override background-image and margin-right is needed for all.